cms/djangoapps/contentstore/utils.py

```diff
@@ -181,6 +181,17 @@ def get_pages_and_resources_url(course_locator):
             pages_and_resources_url = f'{mfe_base_url}/course/{course_locator}/pages-and-resources'
     return pages_and_resources_url
 
+def get_video_upload_url(course_locator):
```

Comment:
```suggestion
def get_mfe_video_upload_url(course_locator):
```
I think it's better to be a little bit more explicit here, since it's hard to distinguish between `video_url` and `video_upload_url` without more context.